source/_integrations/swiss_public_transport.markdown

```diff
@@ -17,7 +17,7 @@ ha_integration_type: integration
 
 The `swiss_public_transport` sensor will give you the next three departure times from a given location to another one in Switzerland.
 
-The [Swiss public transport API](https://transport.opendata.ch/) only allows 1000 requests per 24 hours.
+The [Swiss public transport API](https://transport.opendata.ch/) only allows 1000 requests per 24 hours. The default polling rate is set to `180s`, which works for two connections set up in parallel. If more entries are needed, consider deactivating the default polling for this integration and use an automation to update of the connection using the service `homeassistant.update_entity` and the `time_pattern` trigger.
```

Comment:
Let's just point/link to the chapter below instead and remove mentions of patterns and service identifiers.